### ISSUE TITLE
Scraping Content from LightNovel

### ISSUE THREAD

**Issue by singhayushh** [ProjectSakura/Mirai#9]
### Issue type
Feature addition / enhancement

### About the issue
From the site -> https://www.lightnovelpub.com/ , the content for 
- Popular novels
- Recently updated 
- Recently added novels
are to be fetched just like it is done for other websites in the current project.
This would need backend as well as frontend work which can be each broken into separate issues and worked on.

I would like to take up this issue myself. And I need some more info - whether this should be included under any existing page like manga / animelist or should I create this as an entirely new module

**Comment by ArunTeltia** [comment]
Yeah just 15 days ago lightnovelworld change there name to lightnovelpub I was also thinking of scraping the website, If you have time can we have a meeting on how i am thinking of pursuing this, you can also suggest some ideas of you as well

**Comment by singhayushh** [comment]
I have exams going on in my uni till 23rd. Will it be okay if we talk after 23rd?

**Comment by ArunTeltia** [comment]
No problem let me know, we will work on it

**Comment by subhangi2731** [comment]
@singhayushh are u working on this

**Comment by subhangi2731** [comment]
@ArunTeltia if @singhayushh is not working can I work on this??

**Comment by singhayushh** [comment]
I haven't been able to get time off my schedule lately. Sorry for that. 

@subhangi2731 you may take this up if you'd want
